File: content/snippet/code-signal/arcade/intro/dark-wilderness/growing-plant/growing-plant.py
def growing_plant(up_speed, down_speed, desired_height):
    height = 0
    day = 0
    while height < desired_height:
        day += 1
        height += up_speed
        if height >= desired_height:
            break
        height -= down_speed
    return day


print(growing_plant(6, 5, 10))

File: content/snippet/code-signal/arcade/intro/dark-wilderness/growing-plant/growing-plant.js
function growingPlant(upSpeed, downSpeed, desiredHeight) {
  let days = 0;
  let height = 0;
  while (height < desiredHeight) {
    days++;
    height += upSpeed;
    if (height >= desiredHeight) {
      break;
    }
    height -= downSpeed;
  }
  return days;
}

console.log(growingPlant(6, 5, 10));
